packages/terra-infinite-list/src/terra-dev-site/doc/example/RefcallBackExample.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
```

Comment:
Remove this example. This is not a recommended pattern.

Reply:
Removed refCallBackExample.

Reply:
@supreethmr  did the example get removed?